core-services/release-controller/_releases/release-ocp-4.14.json

```diff
@@ -166,6 +166,11 @@
         "name": "periodic-ci-openshift-release-master-ci-4.14-e2e-aws-sdn-techpreview-serial"
       }
     },
+    "aws-ovn-upgrade-rollback": {
+      "prowJob": {
```

Comment:
I think you want an `"optional": true,` in here to catch up with #43370.

Reply:
Good catch! Thank you!